includes/notifications/notifications-send.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Wporg\TranslationEvents\Notifications;
+
+use DateTime;
+use DateTimeZone;
+use Wporg\TranslationEvents\Attendee\Attendee;
+use Wporg\TranslationEvents\Attendee\Attendee_Repository;
+use Wporg\TranslationEvents\Event\Event;
+use Wporg\TranslationEvents\Event\Event_Repository_Interface;
+use WP_User;
+use Wporg\TranslationEvents\Event\Event_Start_Date;
+
+class Notifications_Send {
+
+	private Attendee_Repository $attendee_repository;
+	private Event_Repository_Interface $event_repository;
+
+	/**
+	 * Notifications_Send constructor.
+	 *
+	 * @param Event_Repository_Interface $event_repository    Event repository.
+	 * @param Attendee_Repository        $attendee_repository Attendee repository.
+	 */
+	public function __construct(
+		Event_Repository_Interface $event_repository,
+		Attendee_Repository $attendee_repository
+	) {
+		$this->event_repository    = $event_repository;
+		$this->attendee_repository = $attendee_repository;
+		add_action( 'wporg_gp_translation_events_email_notifications_1h', array( $this, 'send_notifications' ), 10, 1 );
+		add_action( 'wporg_gp_translation_events_email_notifications_24h', array( $this, 'send_notifications' ), 10, 1 );
+	}
+
+	/**
+	 * Send notifications to the attendees of the event.
+	 *
+	 * @param int $post_id Post ID.
+	 *
+	 * @return void
+	 */
+	public function send_notifications( int $post_id ) {
+		$event     = $this->event_repository->get_event( $post_id );
```

Comment:
It's possible that `get_event()` returns `false`, so I think we should check for that. Otherwise the call to `$event->id()` below will result in an error.